Framework/CurveFitting/test/Algorithms/PlotPeakByLogValueTest.h

```diff
@@ -140,7 +140,8 @@ class PlotPeakByLogValueTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(tnames[8], "f1.PeakCentre_Err");
     TS_ASSERT_EQUALS(tnames[9], "f1.Sigma");
     TS_ASSERT_EQUALS(tnames[10], "f1.Sigma_Err");
-    TS_ASSERT_EQUALS(tnames[11], "Chi_squared");
+    TS_ASSERT_EQUALS(tnames[11], "f1.Intensity");
+    TS_ASSERT_EQUALS(tnames[12], "Chi_squared");
```

Comment:
It's probably a good idea to add a check that the intensity value actually got into the table, add more `TS_ASSERT_DELTA(result->Double(0, 0), 1, 1e-10);` type checks below but for the intensity column.